flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -29,12 +29,14 @@ function run_test {
     description="$1"
     command="$2"
 
+    name=$(basename "${command}" | sed -e 's/[_ ]/-/g' | sed -n 's/\.sh//p')
```

Comment:
why not base it on the description instead? This should be more informative as the script is shared for multiple tests.

Reply:
I chose the script name cause the descriptions right now are rather long, usually with common prefix, plus some of them are still same for different test runs e.g. : `Local recovery and sticky scheduling end-to-end test`.

The command contains script parameters as well. So it should be even easier to differentiate runs this way rather than using description.

Example dir name: `temp-test-resume-savepoint-2-2-file-true-34211378012`